src/app/screens/ConfirmPayment/index.test.tsx

```diff
@@ -0,0 +1,40 @@
+// import { render, screen } from "@testing-library/react";
```

Comment:
This keeps failing for reasons mentioned in the PR description.  
Happy to pair with someone on this or remove it for now to get the general setup in first.